modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -461,6 +462,14 @@ API has not been changed, and it should behave identically as it did in previous
 `Title`, `Heading`, `BodyText` and `Subtext`, have not been changed since they extend the `Text`
 component. These changes do not affect the components API.
 
+### Radio
+
+**PR:** [#2546](https://github.com/Workday/canvas-kit/pull/2546)
+
+`Radio` and `RadioGroup` now use Canvas Tokens and our new styling utilities. The
```

Comment:
In my formfield PR, I'm linking to the docs for this, i feel like people will want to know/ read more on this https://github.com/Workday/canvas-kit/pull/2541/files#diff-8e6215e01f756321dcef04add1f3d985039b84c01deab28c6f89be8bfaa098e1R345

Reply:
Perfect. 

Thanks Manny!